atlp-rwanda/hackers-ec-be
# [FIX]: Orders should include detailed product information for the sold items

### What does this PR do?
This PR ensures that when a user retrieves all orders or a single order, they can view comprehensive details about the products they have sold.

### How to test this PR:
1. Clone this repository.
2. Install the required npm packages.
3. Log in as a buyer, purchase a product, and then navigate to the `order` endpoint. You should be able to see information about the product(s) you sold.
